pyproject.toml

```diff
@@ -79,13 +79,13 @@ coloredlogs = ">=10,<16"
 sanic = "^21.12.0"
 typing-extensions = ">=4.1.1,<5.0.0"
 Sanic-Cors = "^2.0.0"
-prompt-toolkit = "^3.0,<3.0.29"
```

Comment:
This update has to be validated with rasa-private CLI commands (like `rasa interactive` or `rasa shell`), because they had introduced a breaking change in `3.0.30` which I'm not sure they addressed.